salttesting/jenkins.py

```diff
@@ -1800,6 +1800,7 @@ def build_default_test_command(options):
         test_command.append(
             '--names-file="{0}\\tests\\whitelist.txt"'
             ''.format(options.package_source_dir))
+        test_command.append('--unit-tests')
```

Comment:
It would probably be cleaner to just change the `.append()` on L1800 to an `.extend()` and then just pass in an iterable.